dotnet/src/Connectors/Connectors.Memory.SqlServer/SqlServerVectorStoreOptions.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.SemanticKernel.Connectors.SqlServer;
+
+/// <summary>
+/// Options for creating a <see cref="SqlServerVectorStore"/>.
+/// </summary>
+public sealed class SqlServerVectorStoreOptions
+{
+    internal static readonly SqlServerVectorStoreOptions Defaults = new();
+
+    /// <summary>
+    /// Gets or sets the database schema.
+    /// </summary>
+    public string? Schema { get; init; } = null;
```

Comment:
Am noting that in PG and Sqlite this isn't nullable, but rather initialized to the default; as discussed before, I think this means that all SQLs always contain the schema (e.g. `public` for PG), whereas we should just omit it like we do here.

@adamsitnik do you mind applying this change (turn Schema to nullable) for the other relation providers as well? Definitely doesn't have to be in this PR.

Reply:
Sure. I've logged an issue to make sure I don't forget to do that: https://github.com/microsoft/semantic-kernel/issues/10853